Automattic/wp-calypso
# Purchases: Add loading placeholder for Edit Card Details page

Fixes #275.

Expected view when page data is loading on mobile:
![screen shot 2015-11-25 at 12 47 16](https://cloud.githubusercontent.com/assets/699132/11396570/b3301998-9372-11e5-8150-eb2dec97433a.png)

and desktop:
![screen shot 2015-11-25 at 12 47 45](https://cloud.githubusercontent.com/assets/699132/11396581/c04821c0-9372-11e5-996d-629f844bf78e.png)

Note that it differs a bit with original wireframe provided by @mikeshelton1503, but I wanted to reflect current view of edit form on this page.

### Testing
1. Go to http://calypso.localhost:3000/purchases.
2. Select purchase paid with credit card.
3. Navigate to `Edit Card Details` page using `Edit Payment Method` link or credit card icon.
4. You should see loading state.

/cc @stephanethomas
